dask_expr/io/parquet.py

```diff
@@ -104,6 +116,11 @@ def _simplify_up(self, parent):
                 kwargs["filters"] = filters.combine(kwargs["filters"]).to_list_tuple()
                 return ReadParquet(**kwargs)
 
+        if isinstance(parent, Lengths):
+            _lengths = self._get_lengths()
+            if _lengths:
+                return Literal(_lengths)
```

Comment:
maybe this?

```suggestion
        super()._simplify_up(parent)
```

(or however super works)

Reply:
Not sure I understand this suggestion? None of `ReadParquet`'s super-classes cover the the `_get_lengths` optimization.

Reply:
I thought you had it on the `IO` class, but maybe this was github's diff misleading me and actually it was on `FromPandas`

Reply:
Ah right, that makes sense. Yeah, I don't want to assume that all IO classes can leverage statistics.